### PR TITLE
DEV-3514-new go exercise reversemenuindex

### PR DESCRIPTION
**Pull request by tiagocollot**
Created and added the solution and test for the new go exercise reversemenuindex

### CODE REVIEW

**Review by jotapero — APPROVED** [review]
Everything in order

**Review by jotapero — APPROVED** [review]

File: tests/reversemenuindex_test/main.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	{"des  rts", "mains ", "  drinks", "starters  "},
+	{"", "desrts", "mins", "drins", "strters"},
+	{" ", "des serts", "ma ins", "drink s", "st arters"},
+	{"desserts", "mai  ns", "dr   inks", "st3arters"},
+	{"desserts", "mains", "drinks", "starters"},
+	{""},
+	{"+55","*@Â£%D", "edr8927", "		| cat -e"},
+	{"741", "852","4","58","87","03","-96"},
+	{"kd","jq","hel$lo","a","B","9W"},
+}
```
Comment (eslopfer) [inline tests/reversemenuindex_test/main.go:23]: This curly brace is not indented, and so aren't the contents of the array. Please run gofmt to make sure this doesn't happen.

Also, it is not and array, it is a slice. The name is misleading.

**Review by eslopfer — CHANGES_REQUESTED** [review]

File: tests/reversemenuindex_test/main.go (hunk 1 above)
Comment (jotapero) [reply tests/reversemenuindex_test/main.go]: fixed

**Review by eslopfer — APPROVED** [review]
LGTM! :rocket:

**Pull request merged**